src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/HtmlCSharp/CompletionResolveHandler.cs

```diff
@@ -75,7 +75,7 @@ internal class CompletionResolveHandler : IRequestHandler<CompletionItem, Comple
 
         public async Task<CompletionItem> HandleRequestAsync(CompletionItem request, ClientCapabilities clientCapabilities, CancellationToken cancellationToken)
         {
-            if (request?.Data == null)
+            if (request.Data is null)
```

Comment:
This file isn't nullable enabled, so the removal of the question mark could introduce a NullReferenceException. `request?.Data is null` would work though.

Reply:
`request` is not nullable even with nullable enabled so we should be good.